apps/bookmark/style/save.css

```diff
@@ -69,10 +69,21 @@ body[data-mode="put"] #edit-bookmark-header {
   text-transform: uppercase;
 }
 
+#app-icon, #app-icon-placeholder {
+  display: block;
+  width: 6rem;
+  height: 6rem;
+  margin: 0.5rem auto 0 auto;
+}
+
+#app-icon.hidden, #app-icon-placeholder.hidden {
```

Comment:
Unfortunately that wouldn't have enough specificity to have any effect, but I've moved them all into one rule including the IDs.